docs/deploy.md

```diff
@@ -121,4 +121,4 @@ Remember to also delete the S3 bucket that was created to hold the application's
 
 ## Learn more
 
-Read more about `sam deploy` in [the official documentation](https://github.com/awslabs/aws-sam-cli/blob/develop/docs/deploying_serverless_applications.md).
```

Comment:
The current URL seems more informative than the one you are suggesting though?

Reply:
Oh weird. When I tried it earlier, it gave me a 404 page. That's why I updated it.